sale_invoicing_date_selection/tests/test_sale_invoicing_date_selection.py

```diff
@@ -24,6 +24,8 @@ def _create_sale_order(self, partner):
 
     def test_andvance_invoice(self):
         self.sale_order_1 = self._create_sale_order(self.partner)
+        for line in self.sale_order_1.order_line:
+            line.product_id_change()
```

Comment:
This shouldn't be needed created things with `Form`. Why do you need it?

Reply:
@pedrobaeza sorry add this for a local problem, but I forgot to remove it. I pushed the change, please review